packages/cli/src/zosfiles/create/vsam/vsam.handler.ts

```diff
@@ -30,6 +30,7 @@ export default class VsamHandler extends ZosFilesBaseHandler {
             dataclass: commandParameters.arguments.dataClass,
             retainFor: commandParameters.arguments.retainFor,
             retainTo: commandParameters.arguments.retainTo,
+            showAttributes: commandParameters.arguments.showAttributes || commandParameters.arguments.attributes,
```

Comment:
Hey @ATorrise,
This is basically the same as my previous comment.

`showAttributes: commandParameters.arguments.attributes,` (https://github.com/zowe/zowe-cli/pull/1981#discussion_r1445291161)